InnoGotchi.MVC/InnoGotchi.MVC/Controllers/FarmController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using AutoMapper;
+using InnoGotchi.MVC.Contracts.Services;
+using InnoGotchi.MVC.Models.Farm;
+using InnoGotchi.MVC.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+
+namespace InnoGotchi.MVC.Controllers
+{
+    public class FarmController : Controller
+    {
+        private readonly IFarmService _farmService;
+        private readonly IPetService _petService;
+        private readonly IMapper _mapper;
+
+        public FarmController(IFarmService farmService, IPetService petService, IMapper mapper)
+        {
+            _farmService = farmService;
+            _petService = petService;
+            _mapper = mapper;
+        }
+
+        public async Task<IActionResult> FarmOverview()
+        {
+            var jwt = Request.Cookies["jwt"];
```

Comment:
extract this two lines in a separate method as they're repeated

Reply:
made